pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -14,7 +14,105 @@
 
 package mixer
 
-// DoNothing holder file.
-func DoNothing() {
-	// do nothing.
+import (
+	xdsapi "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	"github.com/envoyproxy/go-control-plane/envoy/api/v2/listener"
+	http_conn "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/http_connection_manager/v2"
+
+	"istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pilot/pkg/networking/plugin"
+	"istio.io/istio/pilot/pkg/networking/util"
+	"istio.io/istio/pilot/pkg/proxy/envoy/v1"
+)
+
+type MixerPlugin struct{}
+
+// OnOutboundListener implements the Callbacks interface method.
+func (m *MixerPlugin) OnOutboundListener(in *plugin.CallbackListenerInputParams, mutable *plugin.CallbackListenerMutableObjects) error {
+	env := in.Env
+	node := in.Node
+	service := in.Service
+	proxyInstances := in.ProxyInstances
+
+	if service.MeshExternal {
+		return nil
+	}
```

Comment:
this is incorrect. we need this buildMixerHTTPFilter for external services (except the implementation of that filter has issues today).
so, its like buildMixerHTTPFilter(..., service.MeshExternal)

Reply:
i was confused about that - originally it was not added unless MeshExternal was set to true but that seemed the opposite of what we want. changing it back...